gradle/build.go

```diff
@@ -96,9 +97,9 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		command = filepath.Join(context.Layers.Path, d.Name(), "bin", "gradle")
 	} else if err != nil {
 		return libcnb.BuildResult{}, fmt.Errorf("unable to stat %s\n%w", command, err)
-	} else {
+	} else if fi.Mode() != fs.FileMode(0755) {
```

Comment:
What do you think about just a check if the `+x` attibute is already set?

I think that in a git repository, the `gradlew` script usually should have the executable bit set?

Reply:
Honestly, that was my first idea. But didn't know how to do that in Golang. But would be cleaner IMHO.

Reply:
Yeah I know, but the code called to fix will do chmod 0755.

Reply:
I don't think we need to actually care about the permissions in advance. We can run the `chmod`. 

If it fails, we're presently returning an error but that just needs to be changed to emit a WARNING message and proceed.

Reply:
Ok, thanks @dmikusa-pivotal !

Reply:
> If it fails, we're presently returning an error but that just needs to be changed to emit a WARNING message and proceed (i.e. don't return).

That's exactly what the code in PR is meant to do. Who can review it? Code owner is needed as I see.